src/main/resources/application.properties

```diff
@@ -82,3 +82,8 @@ nocNotification.template.tribunal.id=${NOC_NOTIFICATION_TEMPLATE_TRIBUNAL_ID:tes
 
 assign_case_access_api_url=${AAC_URL:http://localhost:4454}
 apply_noc_access_api_assignments_path=/noc/apply-decision
+
+sendNotification.template.id=${SEND_NOTIFICATION_TEMPLATE_ID}
```

Comment:
would recommend having localhost values as well as environment variable references

Reply:
are we sure this is a good idea for `application.properties`? I thought it better to crash on startup if that variable isn't present rather than silently failing to send an email at runtime. For local testing it'll be overridden from `application-dev/cftlib.properties` anyway